f2/server.js

```diff
@@ -54,10 +58,13 @@ const uploadData = async (req, res, fields, files) => {
   // Get all the data in the format we want, this function blocks because we need the data
   var data = await getData(fields, files)
 
-  // Await here because we also need these results before saving
-  await scanFiles(data)
+  // Check file size
+  if (checkFileSize(files)) {
+    // Await here because we also need these results before saving
+    await scanFiles(data)
 
-  contentModeratorFiles(data, () => save(data, res))
+    contentModeratorFiles(data, () => save(data, res))
+  }
```

Comment:
Running locally It hangs if I add a file - I think we need some sort of 
```
else {
  res.statusCode = 502
   res.statusMessage = 'Error: files too big'
   res.send(res.statusMessage)
}
```
here

Reply:
Fixed!